modules/media/help/Setup.md

```diff
@@ -8,20 +8,21 @@ Create the directory on the server.
 The upload path must be readable and writable by your web server; 
 either the web server `user` or `group` must have read and write permissions.
 
-The default group for your web server process is listed below
+The default group for your web server process is listed below:
 ```
 Ubuntu: $GROUP$ = www-data
 CentOS: $GROUP$ = apache
 ```
 
-To find the `user` of your web server, run 
+To find the `user` of your web server, run:<br/> 
```

Comment:
@driusan i think this was merged prematurely. this file should be in markdown, not html